manifests/1/16Software(www/16software/com)/Breevy/4.10/16Software(www.16software.com).Breevy.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: 16Software(www.16software.com).Breevy
+PackageVersion: "4.10"
+PackageLocale: en-US
+Publisher: 16 Software (www.16software.com)

```

```suggestion
Publisher: 16 Software
```
This should match control panel